src/components/Tag/Tag.less

```diff
@@ -3,74 +3,65 @@
 
 .lucid-Tag {
 	display: inline-flex;
-	margin: @size-XXS @size-XXS 0 0;
+	margin: 0 @size-XXS @size-XXS 0;
 
-	&.lucid-Tag-leaf {
-		min-height: @size-tag;
+	&&-is-removable&-leaf {
+		.lucid-Tag-inner {
+			border-color: @featured-color-primary-borderColorMedium;
+		}
 	}
 
-	.lucid-Tag-inner {
+	& &-inner {
 		border-radius: @size-borderRadius;
 		background: @color-lightGray;
 		border: 1px solid @color-gray;
 		font-weight: bold;
 		font-size: @size-font;
 		color: @color-textColor;
 		width: 100%; // fix for IE 11 bug flex wrapping bug
+		padding: @size-XS @size-XS @size-XXS;
+		display: inline-flex;
+		flex-wrap: wrap;
+		align-items: center;
 
-		.lucid-Tag-inner-content {
-			margin: 0 @size-XS;
-			padding: @size-XXS 0;
-			display: flex;
-			align-items: center;
+		.lucid-Tag-inner-children {
+			padding: 0 @size-XS @size-XXS 0;
+		}
 
-			.lucid-Tag-remove {
-				margin-left: auto;
-				padding-left: @size-XS;
-				display: flex;
-				align-items: center;
+		.lucid-Tag-remove {
+			padding: 1px @size-XS 0 0;
 
-				.lucid-Tag-remove-button {
-					fill: fade(@color-darkGray, 50%);
-				}
+			&-button {
+				fill: fade(@color-darkGray, 50%);
+			}
 
-				&:hover {
-					.lucid-Tag-remove-button {
-						fill: @color-darkGray;
-					}
+			&:hover {
+				.lucid-Tag-remove-button {
+					fill: @color-darkGray;
 				}
 			}
 		}
 
-		.lucid-Tag-subgroup {
-			display: flex;
-			flex-wrap: wrap;
-			padding: 0 @size-XXS @size-XS @size-XS;
-		}
-
 		.lucid-Tag-leaf& {
```

Comment:
the `&` is one of the lesser understood aspects of the LESS preprocessor. It makes our code hard for newcomers to learn. Is this actually adding a lot of value or could we pull it out into a new rule?

```
.lucid-Tag-leaf .lucid-Tag-remove { .... }
```